terraform/09-video-hearing-core/modules/SignalR/init.tf

```diff
@@ -0,0 +1,8 @@
+# Required for This Provider / Module.
+terraform {
+  required_providers {
```

Comment:
do we need a resource for AZ commands or can we just use Null Resources?
Is this well supported? Secure? performance?

Reply:
required for this module.